ably/rest_client_test.go

```diff
@@ -21,38 +21,39 @@ import (
 
 var _ = Describe("RestClient", func() {
 	var (
-		server *httptest.Server
+		srv *httptest.Server
 
-		createMockedRestClient = func(server *httptest.Server) *http.Client {
+		newHTTPClientMock = func(srv *httptest.Server) *http.Client {
 			return &http.Client{
 				Transport: &http.Transport{
-					Proxy: func(req *http.Request) (*url.URL, error) {
-						return url.Parse(server.URL)
-					},
+					Proxy: func(req *http.Request) (*url.URL, error) { return url.Parse(srv.URL) },
 				},
 			}
 		}
 	)
 
-	Context("with a failing request", func() {
+	Context("with a failing req", func() {
```

Comment:
This is supposed to be read as normal english :smile:. You are not supposed to copy variable names in there.